dlt-connector/src/proto/TransactionBody.ts

```diff
@@ -0,0 +1,19 @@
+import { Decimal } from 'decimal.js-light'
+import { TransactionType } from '../graphql/enum/TransactionType'
+import { Field, Message } from '@apollo/protobufjs'
```

Comment:
why using protobuf instead of type-graphql like the other graphql-models of the other modules?

Reply:
I use protobuf for serializing the object before sending it to iota.
- signing is easier
- the iota message is shorter as if I would use json for example (it seems that the Proof-of-Work is harder as longer the message is)
- the iota message on the tangle is harder to read for people without the right software, the transaction from the test would look like this on tangle explorer: 08021202313018b4f5afa506
in json:
```json
{type: 'SEND', amount: '10', createdAt: 1688992436}
```

Reply:
if an additional model will offer more advantages than the mapping to and from this model, than feel free to use it. 
But keep in mind, that an additional mapping and conversion between different models will cost additional resources - memory, cpu, garbagecollection,... - at runtime each time of processing.

Additional efforts during coding phase, because of unlikely inconvinient code, will cause these higher efforts only once for development. At runtime it could be more efficient than a simply looking code, which hides its complexity only from surface.